rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/tests/activityList.spec.ts

```diff
@@ -0,0 +1,363 @@
+import {
+  rundeckServiceMock,
+  rundeckClientMock,
+  axiosMock,
+  i18nMocks,
+  mockEventBus,
+  mockReports,
+} from "../mocks/mock";
+import { flushPromises, shallowMount, VueWrapper } from "@vue/test-utils";
+import ActivityList from "../activityList.vue";
+import ActivityFilter from "../activityFilter.vue";
+import OffsetPagination from "../../../../library/components/utils/OffsetPagination.vue";
+import { cloneDeep } from "lodash";
+import { Modal, Btn } from "uiv";
+jest.mock("../../../../library/rundeckService", () => rundeckServiceMock);
+jest.mock("@rundeck/client", () => rundeckClientMock);
+jest.mock("axios", () => axiosMock);
+const mockRunningExecutions = mockReports.map((report) => report.execution);
+type ActivityListInstance = InstanceType<typeof ActivityList>;
+const shallowMountActivityList = async (
+  props = {},
+  options: Record<string, any> = {}, // Use a more generic type for options
+) => {
+  const wrapper = shallowMount(ActivityList, {
+    props: {
+      eventBus: mockEventBus,
+      displayMode: "mode",
+      ...props,
+    },
+    global: {
+      components: {
+        OffsetPagination,
+        ActivityFilter,
+        Modal,
+        Btn,
+      },
+      stubs: {
+        modal: false,
+        "i18n-t": true,
+        btn: false,
+        ProgressBar: true,
```

Comment:
it shouldn't be necessary to mark a stub as true, please try to remove lines 39 and 41 and double-check if tests are still passing.

Reply:
> it shouldn't be necessary to mark a stub as true, please try to remove lines 39 and 41 and double-check if tests are still passing.

test is passing after removing but giving console.warn